Sources/TokamakCore/DynamicProperty.swift

```diff
@@ -24,34 +24,3 @@ public protocol DynamicProperty {
 extension DynamicProperty {
   public mutating func update() {}
 }
-
-extension TypeInfo {
-  /// Extract all `DynamicProperty` from a type, recursively.
-  /// This is necessary as a `DynamicProperty` can be nested.
-  /// `EnvironmentValues` can also be injected at this point.
-  func dynamicProperties(_ environment: EnvironmentValues,
```

Comment:
This is moted to `MountedElement.swift` to maintain just a single extension there.